samples/k8s_v1_blueprint.yaml

```diff
@@ -9,7 +9,22 @@ metadata:
   name: blueprint-sample
 spec:
   blueprint: |
-    {"blueprintApi":"v2","dogus":[{"name":"official/postgresql","version":"12.15-2","targetState":"present"}]}
+    {
+    "blueprintApi":"v2",
+    "dogus":[
+      {
+        "name":"official/postgresql",
+        "version":"12.15-2",
+        "targetState":"present"
+      }
+    ],
+    "components": [
+      {
+        "name":"k8s/k8s-dogu-operator",
```

Comment:
This is a problem that has been introduced earlier before your PR, but we should really consider adding an extra field for the namespace.

Reply:
Oh, that was the assumption of my development decision we discussed earlier. 

Speaking of namespaces, should we provide a potential k8s cluster namespace at this stage? The discussion of components to target other k8s namespaces than the current EcoSystem namespace may be at hand very soon.

Reply:
Little background for that:
The plan for now was that we do not handle the cluster namespace for now. Only the default namespace of the component is used. Adding a dedicated field for this in the blueprint was considered a design flaw as the blueprint itself should be independent of kubernetes, so that we don't have the same problems like with packages again.

the idea:
Components could be a more general concept in blueprints, so that you can use it for any plattform. You just need to add a name, a version and some generic properties (like the k8s-namespace).

This way you cannot port the blueprint directly to a new plattform as some components will differ but the format could stay the same.

Reply:
and yes, at the beginning of the development i was not aware of namespaces for components, sorry for that. :(

Reply:
No sweat! The blueprint thing was exactly planned as that, as a business artifact. I am glad that it is perceived as that. I'll sort out the component namespace thing